aiomysql/sa/connection.py

```diff
@@ -76,8 +77,18 @@ def execute(self, query, *multiparams, **params):
         if isinstance(query, str):
             await cursor.execute(query, dp or None)
         elif isinstance(query, ClauseElement):
-            compiled = query.compile(dialect=self._dialect)
-            # parameters = compiled.params
+            if self._compiled_cache is not None:
+                key = (self._dialect, query)
```

Comment:
I do not think we need `self. _dialect` as a part of the key, since it is always the same object.

Reply:
agree, a kind of overkill)